feature/sponsors/src/main/java/io/github/droidkaigi/confsched2023/sponsors/SponsorsScreen.kt

```diff
@@ -84,7 +82,9 @@ private fun SponsorsScreen(
 ) {
     val scrollBehavior = TopAppBarDefaults.exitUntilCollapsedScrollBehavior()
     Scaffold(
-        modifier = Modifier.testTag(SponsorsScreenTestTag),
+        modifier = Modifier
+            .nestedScroll(scrollBehavior.nestedScrollConnection)
```

Comment:
How about adding a scroll screenshot test as I mentioned here?
https://github.com/DroidKaigi/conference-app-2023#testing-section-what-to-test

Reply:
Sounds good! I'll give it a try!

Reply:
Thaknks!
I added a scroll screenshot test in this commit, based on the link you introduced.
[added a scroll screenshot test.](https://github.com/DroidKaigi/conference-app-2023/pull/577/commits/1f1dd7ec63b715ac2ded1bd7d50f7b2d4b2a9523)